### PR TITLE
refactor: header icon refactor

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

We're currently using a `` element with a `src` attribute and passing the path to the proper icon for the details view header icon (also on the insights guidance content).

We already have a white version of our heart as a react component (wrapping a svg version of the image).

On this PR:
 - Add blue version of our heart
- Using both, blue and white, react components for the icons on the header

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
   - there is no issue filed
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [x] (UI changes only) Added screenshots/GIFs to description above
   - does not apply as it looks the same
- [x] (UI changes only) Verified usability with NVDA/JAWS **=>** _icon is not being read (before, the img tag had an empty alt attribute, svg's don't have alt property but are not announced neither)_

### CODE REVIEW

File: src/tests/unit/tests/common/components/__snapshots__/header-icon.test.tsx.snap [hunk 1]
```diff
@@ -1,17 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`HeaderIconComponent is high contrast mode enabled: false 1`] = `
-<img
-  alt=""
-  className="header-icon"
-  src="../../icons/brand/white/brand-white-48px.png"
-/>
-`;
+exports[`HeaderIconComponent is high contrast mode enabled: false 1`] = `<BrandWhite />`;
```
Comment (Shobhit1) [inline src/tests/unit/tests/common/components/__snapshots__/header-icon.test.tsx.snap]: Is there a way to make this more detailed? As in, if I change something inside the svg for `BrandWhite` and dont change the name of this component, this snapshot will still pass. So, this is not much  helpful imo.

Comment (dbjorge) [reply src/tests/unit/tests/common/components/__snapshots__/header-icon.test.tsx.snap]: IMO, that's a feature, not a bug; if we care to defend against regressions in the svg for BrandWhite, that should be in a brand-white.test.tsx, not in a test of its consumer.

Comment (smoralesd) [reply src/tests/unit/tests/common/components/__snapshots__/header-icon.test.tsx.snap]: Not really, using debug yields a very similar result. Also, look at the component and the original test... it was showing a `` tag (with some attributes). The change yield an equivalent result, except the icon components don't have any attribute.

File: src/icons/brand/blue/brand-blue.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../../common/react/named-sfc';
+
+export const BrandBlue = NamedSFC('BrandBlue', () => (
+    <svg className="header-icon" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (dbjorge) [inline src/icons/brand/blue/brand-blue.tsx]: My understanding is that the correct/standard-compliant equivalent of `alt=""` would be `role="presentation"` on the svg element (or if you wanted it to be non-presentational, to use `role="img"` and add a `<title>Accessibility Insights logo icon</title>` as the first child of the `<svg>` element).

Comment (smoralesd) [reply src/icons/brand/blue/brand-blue.tsx]: We don't want to announce the icon... that's why the original `` tag had `alt=""`

Comment (smoralesd) [reply src/icons/brand/blue/brand-blue.tsx]: I wasn't able to remove auto-merge label before the bot completed this so, PR #703 have a fix for this comment

**Pull request merged**